ansibullbot/triagers/ansible.py

```diff
@@ -357,6 +352,13 @@ def get_rate_limit(self):
         return self.gh.get_rate_limit().raw_data
 
     def run(self):
+        # We support running a single bot for multiple repos/collections, so iterate on each run
```

Comment:
ooh, very cool.
Need to make sure that a separate cache and temp directory is used for each repo.